serving/pom.xml

```diff
@@ -155,10 +155,10 @@
       <optional>true</optional>
     </dependency>
 
-    <!--compile io.github.lognet:grpc-spring-boot-starter:3.0.2'-->
     <dependency>
-      <groupId>io.github.lognet</groupId>
-      <artifactId>grpc-spring-boot-starter</artifactId>
+        <groupId>net.devh</groupId>
+        <artifactId>grpc-server-spring-boot-starter</artifactId>
+        <version>${grpc.spring.boot.starter.version}</version>
```

Comment:
Do we really need to set this? Shouldnt this be set in the parent-pom?

Reply:
This was set as the interceptors do not work properly under the `lognet` grpc starter.

Reply:
Removed version part to be set in the parent POM.